src/data/accessibility_de.json

```diff
@@ -0,0 +1,55 @@
+            "Wir arbeiten an der baldmöglichsten Beseitigung der aufgelisteten Barrieren.",
+            "<b>Datum der Erstellung bzw. der letzten Aktualisierung der Erklärung</b>",
+            "Diese Erklärung wurde am 21.02.2022 erstellt."
+        ]
+    },
+    "section-ios-compatibility-level": {
+        "title": "Stand der Vereinbarkeit mit den Anforderungen",
+        "textblock": [
+            "Die Anforderungen der Barrierefreiheit für mobile Anwendungen ergeben sich aus der BITV 2.0 in ihrer jeweils gültigen Fassung, die auf der Grundlage von § 12d BGG erlassen wurde.",
+            "Die Überprüfung der Einhaltung der Anforderungen der Corona-Warn-App (iOS) beruht auf einer am 18.01.2022 durchgeführten Selbstbewertung.",
+            "<b>Die nachstehend aufgeführten Inhalte der Corona-Warn-App (iOS) sind aus folgenden Gründen nicht barrierefrei:</b><ul><li>Der Inhalt der App ist entgegen der Anforderung 1.3.4 WCAG 2.1 bzw. 11.1.3.4 EN 301 549 V3.2.1 auf die Anzeige im Hochformat beschränkt.</li><li>Das dargestellte Kontrastverhältnis bei der visuellen Darstellung von Text und Bildern von Text im Falle der Beschriftungstexte der Navigationsleiste von der Anforderung 1.4.3 WCAG 2.1 bzw. 11.1.4.3 EN 301 549 V3.2.1 ab.</li><li>Bei Vergrößerung von Text (ohne assistierende Technik um bis zu 200 %) kann es entgegen der Anforderung 1.4.4 WCAG 2.1 bzw. 11.1.4.4.1 EN 301 549 V3.2.1 zur fehlenden Vergrößerung oder zum Abschneiden von Textfeldern kommen.</li><li>Das Nachstehende zur Funktionalität gilt für an das Endgerät angeschlossene, physische Tastaturen. Entgegen der Anforderung 2.1.1 WCAG 2.1 bzw. 11.2.1.1.1 EN 301 549 V3.2.1 ist keine Tastaturnutzung auf der Seite „Nutzungsbedingungen“ sowie „Datenschutz“ möglich. Auf den Seiten „Personen bearbeiten“ und „Orte bearbeiten“ können Inhalte zudem nicht mittels Tastaturnutzung gelöscht werden.</li><li>Auf den Seiten „Personen bearbeiten“ und „Orte bearbeiten“ kann die Löschtaste entgegen Anforderung 2.5.1 WCAG 2.1 bzw. 11.2.5.1 EN 301 549 V3.2.1 nicht mit einer einzelnen Zeigeraktion freigelegt werden.</li></ul>",
+            "Wir arbeiten an der baldmöglichsten Beseitigung der aufgelisteten Barrieren.",
+            "<b>Datum der Erstellung bzw. der letzten Aktualisierung der Erklärung</b>",
+            "Diese Erklärung wurde am 21.02.2022 erstellt."
+        ]
+    },
+    "section-report-barriers": {
+        "headline": {
+        "title": "Barrieren melden: Kontakt zu den Feedback Ansprechpartnern",
```

Comment:
"Feedback Ansprechpartnern" sounds strange.

Reply:
I guess this was translated from English "Feedback contacts".

Reply:
Changed to "Barrieren melden: Kontakt zu den Ansprechpartnern" in https://github.com/corona-warn-app/cwa-website/pull/2508/commits/1801a9ce56cbb3586bc31edafb20eb9a3343cc8a, thanks for the suggestion!